test/testdata/exhaustive.go

```diff
@@ -0,0 +1,17 @@
+//args: -Eexhaustive
```

Comment:
Should another test case including enabling `DefaultSignifiesExhaustive` be added?

Reply:
Yes, that would help. I'll add one.